File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
    const arr = new Array(52).fill(0);
    my_string.split('').forEach((v, i) => {
        const index = alphabet.split('').indexOf(v);
        arr[index]++;
    })
    return arr;
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    arr = [0 for i in range(52)]
    alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
    for v in my_string:
        arr[alphabet.index(v)] += 1
    return arr